tests/integration/synapseclient/models/async/test_json_schema_async.py

```diff
@@ -0,0 +1,576 @@
+        project_model: Optional[Project] = None,
+        file_fixture: Optional[File] = None,
+        table_fixture: Optional[Table] = None,
+        entity_view_fixture: Optional[EntityView] = None,
+        name_suffix: str = "",
+    ) -> Union[Project, Folder, File, Table, EntityView]:
+        """Helper to create different entity types with consistent naming"""
+        entity_name = str(uuid.uuid4()) + name_suffix
+
+        if entity_type == Project:
+            entity = await Project(name=entity_name).store_async()
+        elif entity_type == Folder:
+            folder = Folder(name=entity_name)
+            entity = await folder.store_async(parent=project_model)
+        elif entity_type == File:
+            file_fixture.name = entity_name
+            entity = await file_fixture.store_async(parent=project_model)
+        elif entity_type == Table:
+            table_fixture.name = entity_name
+            entity = await table_fixture.store_async()
+        elif entity_type == EntityView:
+            entity = await entity_view_fixture.store_async()
+        else:
+            raise ValueError(f"Unsupported entity type: {entity_type}")
+
+        self.schedule_for_cleanup(entity.id)
+        return entity
+
+    @pytest.fixture(scope="function")
+    def create_test_organization_with_schema(
+        self, syn: Synapse
+    ) -> Generator[Tuple[JsonSchemaOrganization, str], None, None]:
+        """Create a test organization for JSON schema functionality."""
+        org_name = "dpetest" + uuid.uuid4().hex[:6]
+
+        js = syn.service("json_schema")
+        created_org = js.create_organization(org_name)
+
+        # Add a JSON schema
+        schema = {
+            "$schema": "http://json-schema.org/draft-07/schema#",
+            "$id": "https://example.com/schema/productschema.json",
+            "title": "Product Schema",
+            "type": "object",
+            "properties": {
+                "productId": {
+                    "description": "The unique identifier for a product",
+                    "type": "integer",
+                    "const": 123,
+                },
+                "productName": {
+                    "description": "Name of the product",
+                    "type": "string",
+                    "const": "default product name",
+                },
+                "productDescription": {
+                    "description": "description of the product",
+                    "type": "string",
+                },
+                "productQuantity": {
+                    "description": "quantity of the product",
+                    "type": "integer",
+                },
+            },
+        }
+        test_org = js.JsonSchemaOrganization(org_name)
+        created_schema = test_org.create_json_schema(schema, TEST_SCHEMA_NAME, "0.0.1")
+        yield test_org, created_schema.uri
+
+        js.delete_json_schema(created_schema.uri)
+        js.delete_organization(created_org["id"])
```

Comment:
Any cleanup needs to occur in a `try/finally` block

Reply:
Got it. I added the try/finally block